### ISSUE TITLE
List current materials at start of project

### ISSUE THREAD

**Issue by polyester** [plone/volto-documentation#1]
- https://docs.voltocms.com/ (location of source: https://github.com/plone/volto/tree/master/docs, generated with MkDocs)
- Volto Training (ReST) ([github](https://github.com/plone/training/tree/master/volto), [html](https://training.plone.org/5/volto/index.html))
- React training (ReST) ([github](https://github.com/plone/training/tree/master/react), [html](https://training.plone.org/5/react/index.html))
- Volto theme hands-on theminig (ReST): https://2019.ploneconf.org/training/volto-theming-hands-on-building-plone-org-in-volto/

**Comment by tisto** [comment]
@polyester @svx any particular reason why we created a new repo instead of working on the Volto repo where the current docs are?

**Comment by polyester** [comment]
@tisto, I thought this could be a 'work in progress' one, whereas when Cris and us are messing in the original one that might throw all kinds of errors because that's currently auto-built.

Also: to have issue-tracker here separate for the GSOD project, without it mingling with Volto-as-code-issue-tracker.

I may be wrong there, I have no strong preference, but my (maybe naive) idea was to develop the new documentation here, and once it's in a workable/buildable state we transplant it back.

**Comment by svx** [comment]
Yes also because IMHO it is easier to start with a clean setup and then we take the docs over bit by bit.
That make things like structure, etc easier

**Comment by tisto** [comment]
@polyester @svx I am ok with anything. I was just wondering. Though, at the end I'd prefer to have the docs in the volto repo.
